File: python/problem717.py
#from solution
class Solution:
    def isOneBitCharacter(self, bits):
        """
        :type bits: List[int]
        :rtype: bool
        """
        k=0
        while k <len(bits)-1:
            k+=bits[k]+1
        return k==len(bits)-1

File: swift/problem717.swift
class Solution {
    func isOneBitCharacter(_ bits: [Int]) -> Bool {
        var index = 0
        let length = bits.count - 1
        while (index < length) {
            index += (bits[index] == 1) ? 2 : 1
        }
        return (index == length) ? true : false
    }
}
